kodilla-jdbc/src/main/resources/kodilla-task-16-4.sql

```diff
@@ -0,0 +1,12 @@
+SELECT I.SUMMARY AS TASKS_LIST, L.NAME AS NAME_OF_TASKS, I.ISSUESLIST_ID, L.ID
+FROM ISSUES I
+JOIN ISSUESLISTS L ON I.ISSUESLIST_ID = L.ID;
+
+SELECT I.SUMMARY AS TASKS_LIST, U.FIRSTNAME AS FIRSTNAME, U.LASTNAM AS LASTNAME
+FROM ISSUES I
+JOIN USERS U ON I.ISSUESLIST_ID= U.ID;
+
+SELECT U.FIRSTNAE, U.LSATNAME, COUNT(*), I.SUMMARY
```

Comment:
literówki

Reply:
2. To zapytanie się nie wykona - nie możesz użyć pola `i.summary` bez agregacji, jeśli grupujesz po userze.